cmd/restic/cmd_backup.go

```diff
@@ -115,6 +116,7 @@ func init() {
 	f.BoolVarP(&backupOptions.ExcludeOtherFS, "one-file-system", "x", false, "exclude other file systems")
 	f.StringArrayVar(&backupOptions.ExcludeIfPresent, "exclude-if-present", nil, "takes `filename[:header]`, exclude contents of directories containing filename (except filename itself) if header of that file is as provided (can be specified multiple times)")
 	f.BoolVar(&backupOptions.ExcludeCaches, "exclude-caches", false, `excludes cache directories that are marked with a CACHEDIR.TAG file. See https://bford.info/cachedir/ for the Cache Directory Tagging Standard`)
+	f.StringVar(&backupOptions.ExcludeLargerThan, "exclude-larger-than", "", "max size of the files to be backed up")
```

Comment:
Please change the description to the following, which makes the help text show `size` instead of `string` for the value placeholder:

~~~
"max `size` of the files to be backed up"
~~~